code/modules/mining/mine_items.dm

```diff
@@ -307,11 +321,11 @@
 	throw_force = 7.0
 	w_class = WEIGHT_CLASS_SMALL
 
-/obj/item/shovel/spade/bone
```

Comment:
Why are we getting rid of the bone spade completely?

Reply:
Fixed this was removed by accident